build.gradle.kts

```diff
@@ -210,7 +210,8 @@ tasks.register("javaPostCommit") {
 tasks.register("javaPostCommitSickbay") {
   dependsOn(":runners:samza:validatesRunnerSickbay")
   dependsOn(":runners:flink:validatesRunnerSickbay")
-  dependsOn(":runners:spark:validatesRunnerSickbay")
+  dependsOn(":runners:spark:3:validatesRunnerSickbay")
+  dependsOn(":runners:spark:2:validatesRunnerSickbay")
```

Comment:
Can you also add what we have in the above javaPostCommit?

These, but the sickbay versions:

```
  dependsOn(":sdks:java:extensions:google-cloud-platform-core:postCommit")
  dependsOn(":sdks:java:extensions:zetasketch:postCommit")
  dependsOn(":sdks:java:io:google-cloud-platform:postCommit")
  dependsOn(":sdks:java:io:amazon-web-services:integrationTest")
  dependsOn(":sdks:java:io:amazon-web-services2:integrationTest")
  dependsOn(":sdks:java:extensions:ml:postCommit")
```